component/otelcol/processor/attributes/attributes_test.go

```diff
@@ -445,7 +530,7 @@ func Test_Hash(t *testing.T) {
 					},
 					{
 						"key": "user.email",
-						"value": { "stringValue": "36687c352204c27d9e228a9b34d00c8a1d36a000" }
+						"value": { "stringValue": "0925f997eb0d742678f66d2da134d15d842d57722af5f7605c4785cb5358831b" }
```

Comment:
Apparently this was caused by a breaking [change](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/22831) in Otel. I'll document this one and other breaking changes in our changelog and docs.

Reply:
What impact does the hash algorithm change have on users?

Reply:
Users of `otelcol.processor.attributes` who have a config with `action = "hash"` will see a change in the hash inside the attribute, for the same input data. This may or may not be a problem for them depending on their use case. I will mention this is the CHANGELOG and upgrade guide.